psmc_check.py

```diff
@@ -211,6 +213,14 @@ def main(opt):
                 plots_validation=plots_validation)
 
 
+def calc_off_nom_rolls(states):
+    off_nom_rolls = []
+    for state in states:
+        att = [state[x] for x in ['q1', 'q2', 'q3', 'q4']]
+        time = (state['tstart'] + state['tstop']) / 2
```

Comment:
I think that this is code that @taldcroft added originally for 1DPAMZT, so he should comment (unless someone else did).